.github/workflows/tests.yml

```
@@ -0,0 +1,44 @@
+name: tests
+
+on:
+  push:
+
+jobs:
+  rspec:
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby_version: ["2.6", "2.7", "3.0", "3.1", "3.2", "3.3"]
```

```suggestion
        ruby_version: ["3.0", "3.1", "3.2", "3.3"]
```
Coming back to this, I think it's fair to drop 2.x. Even 3.0 [has now reached EOL](https://endoflife.date/ruby), so we can just support the latest dropped version.